packages/one-bx-shell-app/src/app/app.module.ts

```diff
@@ -31,12 +32,10 @@ import {FormsModule} from '@angular/forms';
     FundamentalNgxCoreModule,
     BrowserAnimationsModule,
     AppShellModule.forRoot('assets/config/plugins.json', true),
-
-
   ],
   declarations: [
     AppComponent,
-    LandingComponent
+    ...COMPONENTS
```

Comment:
this is very bad approach, it was popular some time ago, but now it's a bad practice, mostly because of stuff related to tree shaking

Reply:
inline components, as far as I know long term goal is to detect unused components at build time